backend/src/bundles/users/user.repository.ts

```diff
@@ -110,6 +113,47 @@ class UserRepository implements Repository {
             .execute();
     }
 
+    public async update(
+        query: Record<string, unknown>,
+        payload: Record<string, Partial<UserDetailsModel>>,
```

Comment:
Maybe it's worth to create `toProfileUpdate` or so method in entity, simirarly as for creation?